addons/waterways/river_gizmo.gd

```diff
@@ -213,17 +422,42 @@ func commit_handle(gizmo: EditorSpatialGizmo, index: int, restore, cancel: bool
 	ur.add_undo_property(river, "valid_flowmap", river.valid_flowmap)
 	ur.add_undo_method(river, "update_configuration_warning")
 	ur.commit_action()
-	
+
+	_extra_lines = {}
```

Comment:
I don't think _extra_lines is a very good variable name. Maybe _height_offset_lines?